src/content/en/tools/chrome-user-experience-report/changelog.md

```diff
@@ -0,0 +1,156 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+
+{# wf_updated_on: 2020-01-15 #}
+{# wf_published_on: 2019-01-15 #}
+{# wf_blink_components: N/A #}
+
+# Chrome User Experience Report {: .page-title }
+
+## Changelog {: #changelog }
```

Comment:
Thanks for pulling this together! Have a bunch of small suggestions, but I think it'll be easier to provide that feedback in the draft Google doc.. I'll followup there.